web/src/core/server/models/user.ts

```diff
@@ -0,0 +1,28 @@
+import { model, models, Schema } from "mongoose";
+
+const userSchema = new Schema({
+  clerkId: {
+    type: String,
+    require: true
+  },
+  username: {
+    type: String,
+    require: true
+  },
+  email: {
+    type: String,
+    require: true
+  },
+  password: {
+    type: String,
+    require: true
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+}, { timestamps: true })
```

Comment:
change `userName` to just `name`, and add `required: true` to email

```suggestion
const userSchema = new Schema({
  clerkId: {
    type: String,
    require: true
  },
  name: {
    type: String,
    require: true
  },
  email: {
    type: String,
    require: true,
    unique: true,
  },
  password: {
    type: String,
    require: true
  },
  createdAt: {
    type: Date,
    default: Date.now
  }
}, { timestamps: true })
```